tests/integration/test_installation.py

```diff
@@ -690,6 +690,10 @@ def test_table_migration_job(
         ],
         filename='azure_storage_account_info.csv',
     )
+    installation.save(
+        [AWSRoleAction('arn:aws:iam::12345:instance-profile/role1', 's3', 'WRITE_FILES', 's3://storage5/*')],
```

Comment:
Why do we need these for azure tests to pass? This is a bug

Reply:
we dont need this, i just tried to reuse the same fixture, but will spilt this logic separately into the azure and was test case